Marlin/src/lcd/e3v2/proui/dwin_popup.h

```diff
@@ -69,11 +69,3 @@ void dwinShowPopup(const uint8_t icon, T amsg1=nullptr, U amsg2=nullptr, uint8_t
   dwinDrawPopup(icon, amsg1, amsg2, button);
   dwinUpdateLCD();
 }
-
-template<typename T, typename U>
-void dwinPopupConfirm(const uint8_t icon, T amsg1, U amsg2) {
-  hmiSaveProcessID(ID_WaitResponse);
-  dwinDrawPopup(icon, amsg1, amsg2, BTN_Confirm);  // Button Confirm
-  dwinUpdateLCD();
-}
```

Comment:
Templated functions like this can be used to do double duty for C strings and F strings (assuming `dwinDrawPopup` is also set up to do both C and F strings in a non-templated way (or does checking with `__typeof__` within the templated function).